### PR TITLE
Add a null check to prevent React app from breaking

### LINKED ISSUE #13: App throwing error on first time clone
Hi

I cloned this repo and started the server. At the initial loads, I see an error. Please refer the screenshot: 
image

I am suspecting this due to an unhandled check for the local storage key of "notification". Can you please confirm if this is reproducible? Or am I missing something?

If this is a genuine case, would like to contribute a small check to the code. Thanks!

### PR DESCRIPTION
**Pull request by bhavin-a**
How to check:

1. Switch to the branch.
2. Run `yarn start` to run the app.
3. Verify that the app shows the message - "You have nothing in your inbox. Make sure you're logged in to Stack Overflow so this inbox populates." and should not break.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,33 +1,45 @@
...
   useEffect(() => {
-    setNotifications(JSON.parse(localStorage.getItem('notifications')));
+    const localStorageNotifications = JSON.parse(
+      localStorage.getItem("notifications")
+    );
+    if (!localStorageNotifications) {
+      setNotifications([]);
+    } else {
+      setNotifications(localStorageNotifications);
+    }
```
Comment (bhavin-a) [inline src/App.js]: This is the major change done. Rest seems like formatting changes.

File: src/App.js [hunk 2]
```diff
@@ -1,33 +1,45 @@
-import React, { useEffect, useState } from 'react';
-import './App.css';
+import React, { useEffect, useState } from "react";
+import "./App.css";
```
Comment (devcshort) [inline src/App.js]: Can you change the double quotes back to single quotes. I'd like to try and keep the styling uniform with the rest of the codebase. Generally I just use double quotes for component props. Other than that, the rest of the changes look great!

Comment (bhavin-a) [reply src/App.js]: Sure @devcshort, let me do that. Thanks for pointing this out!

Comment (devcshort) [reply src/App.js]: Thanks for the update! Merging now 🙂

Comment (bhavin-a) [reply src/App.js]: @devcshort Glad could contribute :smile: 

I missed 2-3 changes for the commas, hope that's ok.

Comment (devcshort) [reply src/App.js]: I think that's fine, it looks great

Comment (bhavin-a) [reply src/App.js]: Cool, thanks!

**Pull request merged**